Radio/index.jsx

```diff
@@ -225,14 +216,200 @@ const Radio = React.createClass({
             </div>
           ]
         })}
+
+        {collapsedOptions && <Collapsible
+          onStartFPSCollection={onStartFPSCollection}
+          onEndFPSCollection={onEndFPSCollection}
+          lowFPS={lowFPS}
+          minimumHeight={49}
```

Comment:
Hm. So, Collapsible now supports a starting height. The magic number is the height of the ExpandLabel, which unfortunately would be rather hard to capture (but I should try, yes, I know. I will)